### PR TITLE
Translate mc docs to Chinese.

### PR DESCRIPTION
**Pull request by dingjs**
Translate mc docs to Chinese.

### CODE REVIEW

File: README_zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+cat      显示文件和对象内容。
+pipe     将一个STDIN重定向到一个对象或者文件或者STDOUT。
+share    生成用于共享的URL。
+cp       拷贝文件和对象。
+mirror   给存储桶和文件夹做镜像。
+find     基于参数查找文件。
+diff     对两个文件夹或者存储桶比较差异。
+rm       删除文件和对象。
+events   管理对象通知。
+watch    箭筒文件和对象的事件。
```
Comment (yihuaf) [inline README_zh_CN.md]: 监控 or 监视？

File: README_zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+```sh
+sudo snap install minio-client --edge --devmode
+```
+
+当一个新的版本发布时，你都会自动更新。
+
+## 通过源码安装
+通过源码安装仅适用于开发人员和高级用户。`mc update`命令不支持基于源码安装的更新通知。请从https://minio.io/downloads/#minio-client下载官方版本。
+
+如果您没有Golang环境，请按照 [如何安装Golang](https://docs.minio.io/docs/how-to-install-golang)。
```
Comment (yihuaf) [inline README_zh_CN.md]: 按照->参照

File: README_zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+cat      cp       events   mb       pipe     rm       share    version
+```
+
+## 了解更多
+- [Minio Client完全指南](https://docs.minio.io/docs/minio-client-complete-guide)
+- [Minio快速入门](https://docs.minio.io/docs/minio-quickstart-guide)
+- [Minio官方文档](https://docs.minio.io)
+
+## 贡献
+Please follow Minio [Contributor's Guide](https://github.com/minio/mc/blob/master/docs/zh_CN/CONTRIBUTING.md)
```
Comment (yihuaf) [inline README_zh_CN.md]: 请遵守

File: docs/zh_CN/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+``mc``欢迎各位有识之士的贡献。为了让大家配合的更默契，我们做出如下约定：
+
+* 尽情的fork项目并修改，我们鼓励pull request来讨论代码的修改。
+    - Fork项目
+    - 创建你的特性分支(git checkout -b my-new-feature)
+    - Commit你的修改(git commit -am 'Add some feature')
+    - Push到你的远端 (git push origin my-new-feature)
+    - 创建一个新的Pull Request
+
+* 如果你有``mc``的更多以来， ``mc``使用[govendor](https://github.com/kardianos/govendor)管理它的依赖。
```
Comment (yihuaf) [inline docs/zh_CN/CONTRIBUTING.md]: 依赖

File: docs/zh_CN/minio-client-complete-guide.md [hunk 5]
```diff
@@ -0,0 +1,785 @@
...
+cat      显示文件和对象内容。
+pipe     将一个STDIN重定向到一个对象或者文件或者STDOUT。
+share    生成用于共享的URL。
+cp       拷贝文件和对象。
+mirror   给存储桶和文件夹做镜像。
+find     基于参数查找文件。
+diff     对两个文件夹或者存储桶比较差异。
+rm       删除文件和对象。
+events   管理对象通知。
+watch    箭筒文件和对象的事件。
```
Comment (yihuaf) [inline docs/zh_CN/minio-client-complete-guide.md]: 监视

**Comment by dingjs** [comment]
@yihuaf  ping.

**Comment by deekoder** [comment]
@harshavardhana  since @yihuaf  has approved PTAL.

**Pull request merged**